corehq/apps/app_manager/templates/app_manager/v2/partials/commcare_settings.html

```diff
@@ -3,91 +3,97 @@
-                        <label data-bind="html: name, attr: {for: inputId}" class="inner-control-label"></label>
-                        <span data-bind="makeHqHelp: { name: name, description: $data.description, format: $data.description_format}, visible: $data.description"></span>
-                    </div>
-                    <div class="col-sm-4">
-                        <span data-bind="template: $data.widget_template || 'CommcareSettings.widgets.' + widget"></span>
-                        <p class="help-block" data-bind="visible: disabledButHasValue">
-                            <span data-bind="text: $data.disabled_txt"></span>
-                            <span data-bind="visible: !$data.disabled_txt">
-                            {% blocktrans %}
-                                Oops!
-                                This setting shouldn't be here.
-                                Could you change it to the default
-                                to make it go away?
-                                Sorry about that.
-                            {% endblocktrans %}
-                            </span>
-                        </p>
-                    </div>
-                    <div class="col-sm-2 control-label">
-                        <div class="pull-left">
-                            <span class="label label-danger" data-bind="
-                                text: $data.warning || $parents[1].warning,
-                                visible: !$data.valueIsLegal()">
-                            </span>
-                            <span class="label label-info" data-bind="visible: $data.preview">{% trans "Preview" %}</span>
-                            <span data-bind="visible: disabledMessage(), css: {'text-warning': !optionOK(), 'text-success': optionOK}">
-                                <i class="fa" data-bind="css: {'fa-arrow-left': !optionOK(), 'fa-check': optionOK}"></i>
-                                <span data-bind="text: disabledMessage()"></span>
-                            </span>
-                        </div>
-                    </div>
-                </div>
+  <div id="settings-save-btn"></div>
+  <div data-bind="foreach: sections" id="app-settings-options" class="panel-group">
+    <div class="panel panel-appmanager" data-bind="visible: isVisible">
+      <div class="panel-heading">
+        <h4 class="panel-title">
+          <a data-toggle="collapse"
+             data-parent="#app-settings-options"
+             data-bind="attr: {href: '#' + id}, css: {collapsed: reallyCollapse}">
+            <i class="fa fa-angle-double-down"></i>
+            <span data-bind="text: title"></span>
+          </a>
+        </h4>
+      </div>
+      <div class="panel-collapse collapse" data-bind="attr: {id: id}, css: {'in': !reallyCollapse()}">
+        <div class="panel-body">
+          <!-- ko foreach: settings -->
+          <!-- ko if: widget !== 'none' -->
```

Comment:
Possible bad merge - the `none` widget was removed in https://github.com/dimagi/commcare-hq/pull/13963